taxi/admin.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from taxi.models import Manufacturer, Car, Driver
+
+
+@admin.register(Manufacturer)
+class ManufacturerAdmin(admin.ModelAdmin):
+    list_display = ('name', 'country')
+    search_fields = ('name', 'country')
+
+
+@admin.register(Driver)
+class DriverAdmin(UserAdmin):
+    list_display = UserAdmin.list_display + ('license_number',)
+    fieldsets = UserAdmin.fieldsets + (
+        ('Additional info', {"fields": ('license_number',)}), )
+    add_fieldsets = UserAdmin.add_fieldsets + (
+        ('Additional info', {"fields": ('license_number',)}), )
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    search_fields = ('model',)
+    list_display = ('manufacturer',)
```

Comment:
It would be better to display more fields in the list view of the admin panel. This will provide more useful information to the admin user. Besides 'manufacturer', you might want to include 'model' and other relevant fields.